### PR TITLE
[WIP] Language Improvements

### PR DESCRIPTION
**Pull request by GrahamCampbell**
TODO:
* [x] Move all static text into the language file (thanks @foxted)
* [x] Check we haven't missed anything or broken anything
* [x] Remove languages with less than 50% of the translations (these can be re-added later)
* [x] Cleanup array alignment

---

TODO: in the next pull request, we need to restore the other languages.

### CODE REVIEW

**Comment by foxted** [comment]
I cloned the repo and switched to the lang branch, but I am having this error: 

```
file_get_contents(/Users/foxted/Sites/cachet/public/build/rev-manifest.json): failed to open stream: No such file or directory
```

Not sure where it's coming from...

**Comment by GrahamCampbell** [comment]
You haven't run gulp.

**Comment by GrahamCampbell** [comment]
https://github.com/cachethq/Cachet/blob/master/docs/setup/install.md#get-a-copy

**Comment by foxted** [comment]
My bad, I forgot! :)

**Comment by GrahamCampbell** [comment]
Don't worry about it. I'm heading off to bed now, so won't respond for a while. Let me know how this goes for you. Thanks. :)

**Comment by GrahamCampbell** [comment]
Working on getting this rebased...

**Comment by foxted** [comment]
Thank you @GrahamCampbell :)

File: app/views/dashboard/metrics/index.blade.php [hunk 1]
```diff
@@ -2,11 +2,11 @@
...
 @section('content')
     <div class="header">
-        <i class="ion ion-stats-bars"></i> {{ trans('cachet.dashboard.metrics') }}
+        <i class="ion ion-stats-bars"></i> {{ trans('dashboard.metrics.metrics') }}
     </div>
     <div class="row">
         <div class="col-sm-12">
-            <h3>Metrics</h3>
+            <h3>{{ trans('dashboard.metrics.metrics') }}</h3>
             <p class='lead'>Eventually this page will show all of the graphs that make up your metrics.</p>
```
Comment (GrahamCampbell) [inline app/views/dashboard/metrics/index.blade.php]: missing translation here

**Comment by foxted** [comment]
I thought this view will be subject to change in the future so I skipped this string. But it can easily be added, probably under the `dashboard.metrics.description` key.

Here is a proposed french translation of this sentence: 

`Cette page montrera éventuellement tous les graphiques qui constituent vos point de mesure.`

**Comment by foxted** [comment]
I skipped the email stuff as well, I assumed it was also a work in progress.

**Comment by jbrooksuk** [comment]
@foxted yeah, emails aren't implemented yet.

File: app/views/dashboard/team/add.blade.php [hunk 2]
```diff
@@ -2,40 +2,42 @@
 
 @section('content')
     <div class="header">
-        <span class='uppercase'>
-            <i class="ion ion-person"></i> {{ trans('cachet.dashboard.user') }}
+        <span class="uppercase">
+            <i class="icons ion-person"></i> {{ trans('dashboard.team.team') }}
```
Comment (GrahamCampbell) [inline app/views/dashboard/team/add.blade.php]: typos here

Comment (GrahamCampbell) [inline app/views/dashboard/team/add.blade.php]: i'll fix in the rebase

File: app/lang/en/cachet.php [hunk 3]
```diff
@@ -10,76 +10,38 @@
...
-            'site_domain'       => 'Site Domain',
-            'show_support'      => 'Show support for Cachet?',
-            'admin_account'     => 'Administrator Account',
-            'admin_username'    => 'Username',
-            'finish_setup'      => 'Complete Setup',
-        ],
-    ],
+    'rss-feed'        => 'RSS Feed',
+    'atom-feed'       => 'Atom Feed',
+    'feed'            => 'Status Feed'
```
Comment (GrahamCampbell) [inline app/lang/en/cachet.php]: missing comma

**Comment by jbrooksuk** [comment]
@GrahamCampbell why do we remove existing languages? I know they're incomplete, but it'll fallback to `en` translation anyway.

**Comment by GrahamCampbell** [comment]
Because they're totally wrong. They can be added one by one in separate pull requests. I'm not waiting any longer for them.

**Comment by GrahamCampbell** [comment]
Rebasing this takes ages.

**Pull request merged**